### PR TITLE
jsk_robot_startup/quadruped_joystick_teleop: support dock/undock command

### PR DESCRIPTION
**Pull request by k-okada**

### CODE REVIEW

**Review by sktometometo — APPROVED** [review]

**Comment by k-okada** [comment]
@sktometometo @tkmtnt7000 added dock supoprt, replacement of https://github.com/sktometometo/jsk_robot/pull/100 ??

File: jsk_robot_common/jsk_robot_startup/src/quadruped_joystick_teleop.cpp [hunk 1]
```diff
@@ -5,6 +5,11 @@
 #include <std_srvs/SetBool.h>
 #include <sound_play/SoundRequest.h>
 
+//#include <algorithm>
+
+template <typename T, size_t N> const T* begin(const T (&a)[N]) { return a; }
```
Comment (sktometometo) [inline jsk_robot_common/jsk_robot_startup/src/quadruped_joystick_teleop.cpp:10]: What is the purpose of this function?

Comment (k-okada) [reply jsk_robot_common/jsk_robot_startup/src/quadruped_joystick_teleop.cpp]: oh, that is for support multiple buttons to send services, like `button_xxx: [0, 1]`. we do not need anymore, so removed.

Comment (sktometometo) [reply jsk_robot_common/jsk_robot_startup/src/quadruped_joystick_teleop.cpp]: I see. Thank you.

**Review by sktometometo — APPROVED** [review]

**Pull request merged**